auth_otp/models.py

```
@@ -0,0 +1,438 @@
+    
+    class Meta:
+        verbose_name = 'درخواست OTP'
+        verbose_name_plural = 'درخواست‌های OTP'
+        ordering = ['-created_at']
+        indexes = [
+            models.Index(fields=['phone_number', 'created_at']),
+            models.Index(fields=['otp_code', 'is_used']),
+            models.Index(fields=['expires_at']),
+        ]
+    
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)